addons/web/static/src/js/chrome/action_manager.js

```diff
@@ -536,7 +536,7 @@ var ActionManager = Widget.extend({
                 var message = _t('A popup window has been blocked. You ' +
                              'may need to change your browser settings to allow ' +
                              'popup windows for this page.');
-                this.do_warn(_t('Warning'), message, true);
+                this.do_warn(false, message, true);
```

Comment:
Now that you this function has a really strange API, why continue to use second argument in this case
->
 this.do_warn(message, sticky=true);

Reply:
We considered changing it, but this function is already deprecated (see its definition) and will be removed as soon as everything will be converted to owl. Changing it as you suggest would break retro-compatibility. It wasn't black or white, but we went for this solution.